apps/src/code-studio/pd/application_dashboard/summary.jsx

```diff
@@ -2,18 +2,45 @@
  * Application Dashboard summary view.
  * Route: /summary
  */
-import React from 'react';
+import React, {PropTypes} from 'react';
 import SummaryTable from './summary_table';
+import Spinner from '../components/spinner';
+import $ from 'jquery';
 
 export default class Summary extends React.Component {
+  static contextTypes = {
+    router: PropTypes.object.isRequired
+  };
+
+  state = {
+    loading: true,
+    applications: null
+  };
+
+  componentWillMount() {
+    $.ajax({
+      method: 'GET',
+      url: '/api/v1/pd/applications',
+      dataType: 'json'
+    })
+    .done(data => {
+      this.setState({
+        loading: false,
+        applications: data
+      });
+    });
+  }
+
   render() {
+    if (this.state.loading) {
+      return <Spinner/>;
+    }
     return (
-      <div>
+      <div className="row">
         <h1>All Regional Partner Applications</h1>
```

Comment:
I know it renders okay - but should the H1 be in this div? The way these elements seem composed, it seems like they should be outside of it